src/Components/FieldTrip/getParentList.js

```diff
@@ -0,0 +1,22 @@
+// import React, { useState, useEffect } from "react";
```

Comment:
@ShotaKiko Did you need this getParentList file?

Reply:
nope good catch. gonna delete it